doc/PREFERENCE.md

```diff
@@ -0,0 +1,48 @@
+## Mark Text Preference
+
+#### General
+
+| Key                 | Type    | Default Value | Description                                                                                                                                                |
+| ------------------- | ------- | ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| autoSave            | Boolean | ture          | Automatically save the content being edited. option value: true, false                                                                                     |
+| autoSaveDelay       | Number  | 3000          | How long do you want to save your document(ms)?3000 ~10000                                                                                                 |
+| titleBarStyle       | String  | csd           | The title bar style. the native option will result in a standard gray opaque title bar. `csd`(macOS only), `custom`, `native`                              |
+| openFileInNewWindow | Boolean | false         | true, false                                                                                                                                                |
```

Comment:
This should be `openFilesInNewWindow` because all files via command-line should be opened in the same window. I'm working on this.